src/enzyme_ad/jax/Passes/LowerJIT.cpp

```diff
@@ -991,13 +998,12 @@ struct LowerJITPass
         replacement = rewriter.create<stablehlo::CustomCallOp>(
             op.getLoc(), op.getResultTypes(), op.getInputs(),
             rewriter.getStringAttr("enzymexla_compile_cpu"),
-            /* has_side_effect*/ rewriter.getBoolAttr(false),
-            /*backend_config*/ backendstr,
+            /* has_side_effect*/ op.getHasSideEffectAttr(),
+            /*backend_config*/ dattr,
             /* api_version*/
             CustomCallApiVersionAttr::get(
                 rewriter.getContext(),
-                mlir::stablehlo::CustomCallApiVersion::
```

Comment:
oh no this is actually an issue, this needs to keep its API as it was before (it is the calling convention for the setup)

Reply:
`API_VERSION_STATUS_RETURNING_UNIFIED` doesn't allow for a DictAttr

Reply:
yeah, but we shouldn't change the attributes passed in

Reply:
if we pass backend_config to jit_call those need to be forwarded to the custom_call as well right?

Reply:
so currently the backend config out of jitcall is the function pointer of the jitt'd function, which then is directly executed